### ISSUE TITLE
HasNoTokens.sol issues  Unused local variable compilation warning

### ISSUE THREAD

**Issue by yaronvel** [OpenZeppelin/zeppelin-solidity#367]
solc version 13.

```
zeppelin-solidity/contracts/ownership/HasNoTokens.sol:21:26: Warning: Unused local variable
  function tokenFallback(address from_, uint256 value_, bytes data_) external {
                         ^-----------^
,zeppelin-solidity/contracts/ownership/HasNoTokens.sol:21:41: Warning: Unused local variable
  function tokenFallback(address from_, uint256 value_, bytes data_) external {
                                        ^------------^
,zeppelin-solidity/contracts/ownership/HasNoTokens.sol:21:57: Warning: Unused local variable
  function tokenFallback(address from_, uint256 value_, bytes data_) external {
                                                        ^---------^
```

**Comment by frangio** [comment]
See #410 for a tentative solution that commented out the variable names. Note that the NatSpec documentation for the variables had to be removed, which is not really acceptable.

The proposed solution is to add a no-op mention of the variables. I think simply adding a statement for each variable works: `from_; value_; data_;`.

**Comment by Shrugs** [comment]
I don't get these errors on 0.4.18 after running `truffle compile`. What is your complication command?

**Comment by frangio** [comment]
True, this was fixed in #576!
